### ISSUE TITLE
Thermostat doesn't report correct `max_temp` or `min_temp` to Home Assistant

### ISSUE THREAD

**Issue by ericlathrop** [zwave-js/zwave-js-ui#3585]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Z-Wave JS UI version

9.8.2

### ZwaveJS version

whatever comes in the 9.8.2 docker image

### Describe the bug

My thermostat (Honeywell T6 Pro) has config values for min/max temperatures:
![image](https://github.com/zwave-js/zwave-js-ui/assets/530891/2dbb0203-7d70-434d-8fd2-69f2b2b4bb97)

However, zwave-js-ui reports a hardcoded inaccurate min/max temperature to HA. See: https://github.com/zwave-js/zwave-js-ui/blob/3e632dbdab5216d8df45b6ef61b05725f988e2ee/api/hass/configurations.ts#L178

This prevents the thermostat card in HA from adjusting the temperature to reasonable values (I use Fahrenheit, and the hardcoded numbers are reasonable for Celsius).

### To Reproduce

1. Include a thermostat on your z-wave network
2. Have zwave-js-ui connected to Home Assistant through MQTT
3. Add a Thermostat card to a HA dashboard
4. Try to adjust the temperature to 70F, and notice it only lets you go to 40F

### Expected behavior

I expect zwave-js-ui to send the min/max temperatures configured in my thermostat to HA.

**Comment by ericlathrop** [comment]
I lied in the checklist about a HA developer sending me here because I'm a developer (not for this project or HA), and I read enough of the code that I believe the issue is in this project.

**Comment by robertsLando** [comment]
@ericlathrop Are you using mqtt discovery or home assistant zwave-js integration? If you are using the first I suggest you to switch to the integration as described [here](https://zwave-js.github.io/zwave-js-ui/#/homeassistant/homeassistant-official), if you are using mqtt I suggest to firstly read the [docs](https://zwave-js.github.io/zwave-js-ui/#/homeassistant/homeassistant-mqtt)

TLDR;

> The preferred method of integrating your Z-Wave components with Home Assistant is through the official Home Assistant [Z-Wave JS integration](https://www.home-assistant.io/integrations/zwave_js), this because MQTT Discovery is limited compared to a native integration and Home Assistant updates frequently break it. Based on this I would NOT RECOMMEND using MQTT Discovery, I don't plan to keep it maintained in the future.

**Comment by ericlathrop** [comment]
Ah dang, I just migrated from old HA with zwave built-in to zwave-js-ui, and thought MQTT was the new hotness.

**Comment by robertsLando** [comment]
@ericlathrop I dunno if I explained correctly but you should still use z-ui, you just have to enable ws server in home assistant settings and set the zwave-js integration to point to z-ui instance

**Comment by ericlathrop** [comment]
@robertsLando I understand what you mean. I suppose that means all my devices will disappear in home assistant, and I'll have to recreate all my dashboards and automations, which is a bunch of work.

I'd suggest putting some sort of "deprecated" warning near the "MQTT Discovery" button in zwave-js-ui.

**Comment by robertsLando** [comment]
I cannot deprecate it as there are still users that rely on it (and don't have an alternative like Home Assistant users). Anyway this is well explained in docs, I always suggest to read docs before jumping into a new application (based also on my personal experience)

**Comment by ericlathrop** [comment]
I did read the docs, but not every page. The warning is on https://zwave-js.github.io/zwave-js-ui/#/homeassistant/homeassistant-official but not on https://zwave-js.github.io/zwave-js-ui/#/homeassistant/homeassistant-mqtt

**Comment by robertsLando** [comment]
@ericlathrop It's in the very first lines of https://zwave-js.github.io/zwave-js-ui/#/homeassistant/homeassistant-mqtt

**Comment by ericlathrop** [comment]
Hahaha, sorry, I guess I'm blind!
